README.md

```diff
@@ -1,9 +1,5 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Getting started
-
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
```

Comment:
This comment isn't untrue. You do still use a virtual environment to isolate package dependencies, but it's handled by Poetry.

What you **should** delete is the section below about setup.sh. And you should delete the setup.ps1 file.